services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -174,10 +174,12 @@ const indexBlock = async job => {
 						block.generatorAddress, blockReward, commissionAmount,
 					);
 
+					logger.trace(`Incrementing validator commission with address ${block.generatorAddress} by ${commissionAmount}.`);
 					await validatorsTable.increment({
 						increment: { totalCommission: BigInt(commissionAmount) },
 						where: { address: block.generatorAddress },
 					}, dbTrx);
+					logger.trace(`Incrementing validator self stake rewards with address ${block.generatorAddress} by ${selfStakeReward}.`);
```

Comment:
Also, please add a similar debug statement below for self-stake rewards.
```suggestion
					logger.trace(`Increasing commission for validator ${block.generatorAddress} by ${commissionAmount}.`);
					await validatorsTable.increment({
						increment: { totalCommission: BigInt(commissionAmount) },
						where: { address: block.generatorAddress },
					}, dbTrx);
					logger.debug(`Increased commission for validator ${block.generatorAddress} by ${commissionAmount}.`);
					logger.trace(`Increasing self-stake rewards for validator ${block.generatorAddress} by ${commissionAmount}.`);
```